src/components/bookmarkV2/index.js

```diff
@@ -45,11 +53,21 @@ class Bookmark extends React.Component {
     const accountsLength = this.getFilterList().length;
 
     // istanbul ignore else
-    if (action === 'down' && dropdownIndex < accountsLength - 1) this.setState({ dropdownIndex: dropdownIndex + 1 });
-
+    if (action === 'down' && dropdownIndex < accountsLength - 1) {
+      if (dropdownIndex + 1 >= 4) {
+        this.listContainerRef.scrollTop = this.listContainerRef.scrollTop + 44;
+      }
+      this.setState({ dropdownIndex: dropdownIndex + 1 });
+    }
 
     // istanbul ignore else
-    if (action === 'up' && dropdownIndex > 0) this.setState({ dropdownIndex: this.state.dropdownIndex - 1 });
+    if (action === 'up' && dropdownIndex > 0) {
+      this.listContainerRef.scrollTop = this.listContainerRef.scrollTop > 0
+        && (dropdownIndex - 1) * 44 < this.listContainerRef.scrollTop
+        ? this.listContainerRef.scrollTop - 44
+        : this.listContainerRef.scrollTop;
+      this.setState({ dropdownIndex: this.state.dropdownIndex - 1 });
```

Comment:
Since you already got the `dropdownIndex` value from the store, there's no need to use this.state.dropdownIndex

Reply:
done